sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -520,7 +522,7 @@ case class DescribeTableCommand(table: TableIdentifier, isExtended: Boolean, isF
 
   private def describeSchema(schema: Seq[CatalogColumn], buffer: ArrayBuffer[Row]): Unit = {
     schema.foreach { column =>
-      append(buffer, column.name, column.dataType.toLowerCase, column.comment.orNull)
+      append(buffer, column.name, column.dataType.toLowerCase, column.comment.getOrElse(""))
```

Comment:
TBH, I just followed the original logic while rewriting this part and used null here. Technically, null means no comment, which is different from empty comment.